visualization/app/codeCharta/state/effects/unfocusNodesOnLoadingMap.effect.ts

```diff
@@ -0,0 +1,18 @@
+import { Inject } from "@angular/core"
+import { filter, map } from "rxjs"
+import { isActionOfType } from "../../util/reduxHelper"
+import { createEffect } from "../angular-redux/effects/createEffect"
+import { Actions, ActionsToken } from "../angular-redux/effects/effects.module"
+import { IsLoadingFileActions } from "../store/appSettings/isLoadingFile/isLoadingFile.actions"
+import { unfocusAllNodes } from "../store/dynamicSettings/focusedNodePath/focusedNodePath.actions"
+
+export class UnfocusNodesOnLoadingMapEffect {
+	constructor(@Inject(ActionsToken) private actions$: Actions) {}
+
+	unfocusNodesOnLoadingMap$ = createEffect(() => {
+		return this.actions$.pipe(
+			filter(action => isActionOfType(action.type, IsLoadingFileActions)),
```

Comment:
This is the previous behavior. Is this really wanted? At least only unfocus when the payload was set to `true`?